dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/AbstractFullReadOnlyController.java

```diff
@@ -150,6 +157,22 @@
     @Autowired
     protected AttributeService attributeService;
 
+    // --------------------------------------------------------------------------
+    // Custom CSV mapper for field filtering
+    // --------------------------------------------------------------------------
+
+    protected static final CsvMapper CSV_MAPPER = new CsvMapper();
+
+    static
+    {
+        CsvMapper csvMapper = new CsvMapper();
```

Comment:
Perhaps move these lines to separatem documented method to centralize the code bit more?